modules/rtm/R/invert.fast.R

```diff
@@ -33,6 +30,7 @@
 #' @return Matrix (ngibbs x (length(inits) + 1)) of MCMC samples 
 #'          of parameters.
 invert.fast <- function(modname, observed, inits, cons, pmu, psd, plog, minp, ngibbs) {
+  stop('This function is deprecated. Please use `invert.auto`.')
```

Comment:
Perfectly happy to just remove these functions, since they're immortalized in the git history.